zproject/urls.py

```diff
@@ -520,6 +520,11 @@
             {'GET': ('zerver.views.upload.serve_file_backend',
                      {'override_api_url_scheme'})}),
 
+# This endpoint will be used to proxy thumbnail requests to thumbor host in case
+# we are in dev environment or thumbor is not on same server as zulip.
```

Comment:
Just to confirm, this endpoint is used always, whether they're the same server or not, right?